Modules/xASL_module_ASL.m

```diff
@@ -62,8 +62,8 @@
 
 
 %% A. Check if ASL exists, otherwise skip this module
-[x,result,skip] = xASL_module_ASL_CheckASL(x,result);
-if skip, return; end
+[x, bSkip] = xASL_module_ASL_CheckASL(x);
```

Comment:
Not true that `result` is unused and can be deleted. `result` and also `x.result` are output parameters of this function - so they should both be assigned before doing `return`. Please revert and add `x.result`.

Reply:
Fixed this now in [c24f23ad](https://github.com/ExploreASL/ExploreASL/pull/1234/commits/c24f23ad5a6af40ea9d8dc823da17aa900e50fc5). But it seems that `x.results` is not really used, it is initialized in `xASL_module_Import`, `xASL_module_ASL`, `xASL_init_InitializeMutex` on several locations but not interpreted anywhere, so shall I remove it?

Reply:
Also note that you need to add not only `result` here but also `x.result` that was previously missing - just that it's consistent before we decide to remove it (just in case we don't).